WordPress/Classes/Networking/PostServiceRemoteXMLRPC.m

```diff
@@ -164,6 +169,54 @@ - (void)deletePost:(RemotePost *)post
     }
 }
 
+- (void)trashPost:(RemotePost *)post
+          forBlog:(Blog *)blog
+          success:(void (^)(RemotePost *))success
+          failure:(void (^)(NSError *))failure
+{
+    NSParameterAssert([post.postID longLongValue] > 0);
+    NSNumber *postID = post.postID;
+    if ([postID longLongValue] > 0) {
+        NSArray *parameters = [blog getXMLRPCArgsWithExtra:postID];
+
+        WPXMLRPCRequest *deletePostRequest = [self.api XMLRPCRequestWithMethod:@"wp.deletePost" parameters:parameters];
```

Comment:
You can always use the split view, which does wrapping
https://github.com/wordpress-mobile/WordPress-iOS/pull/3608/files?diff=split

I don't think trying to fit Objective-C code to 80 characters is a good idea, considering how it favors verbose naming. To give you an example, something as common as this already breaks the 80 character limit:

```
    NSManagedObjectContext *context = [[ContextManager sharedInstance] newDerivedContext];
```